Fix CI failing to compose-up for 2 reasons

mikedep333 opened the pull request:
Fix CI failing to compose-up the nightly s6 images

because it is looking for the non-existing tags of the development branch's web images. It should use "nightly" instead.

[noissue]




 Workaround CI failing due to an ubuntu podman bug
    
 https://bugs.launchpad.net/ubuntu/+source/libpod/+bug/2024394
    
 Workaround will disable itself when no longer needed
    
 [noissue]

mikedep333 committed:
Fix CI failing to compose-up the nightly s6 images

because it is looking for the non-existing tags of the
development branch's web images. It should use "nightly" instead.

[noissue]

decko commented on .github/workflows/pulp_images.yml:
```diff
@@ -242,7 +242,10 @@ jobs:
+            if [ "${{ matrix.image_variant }}" == "nightly" ]
+              WEB_TAG="nightly"
+            else
+              WEB_TAG="${APP_BRANCH}"
```
I think we need to close this `if` block here.
```suggestion
            if [ "${{ matrix.image_variant }}" == "nightly" ] then
              WEB_TAG="nightly"
            else
              WEB_TAG="${APP_BRANCH}"
            fi
```

decko commented on .github/workflows/pulp_images.yml:
```diff
@@ -242,7 +242,11 @@ jobs:
             FILE="docker-compose.folders.yml"
             # We'll pull the web image from a registry since we didn't build it.
-            WEB_TAG="${APP_BRANCH}"
+            if [ "${{ matrix.image_variant }}" == "nightly" ]
```
I think we're missing a `then` by the end of this line, like this:
```suggestion
            if [ "${{ matrix.image_variant }}" == "nightly" ] then
```

mikedep333 replied:
You forgot the semicolon.

Fixed it :)

mikedep333 committed:
Workaround CI failing due to an ubuntu podman bug

https://bugs.launchpad.net/ubuntu/+source/libpod/+bug/2024394

Workaround will disable itself when no longer needed

[noissue]

mikedep333 commented:
Looks like the 2 remaining CI failures are due to an additional issue.

I'd like to merge, and then fix this next issue.
